src/commands/synthetics/interfaces.ts

```diff
@@ -268,6 +275,9 @@ export interface ConfigOverride {
   followRedirects?: boolean
   headers?: {[key: string]: string}
   locations?: string[]
+  mobileApplication?: MobileApplication
+  mobileApplicationVersion?: string
+  mobileApplicationVersionFilePath?: string
```

Comment:
IIUC we don't need these additional fields, and could us `mobileApplication` object now